app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,3 @@
+class Users::SessionsController < Devise::SessionsController
+  after_action :verify_authorized, except: [:new, :create, :destroy]
```

Comment:
this may change, however, in the future if we decide to de-authorize based on domain.